tests/python_client/testcases/test_search.py

```diff
@@ -3350,6 +3353,39 @@ def test_search_with_comparative_expression(self, _async):
             ids = hits.ids
             assert set(ids).issubset(filter_ids_set)
 
+    @pytest.mark.tags(CaseLabel.L2)
+    def test_search_expression_with_double_quotes(self):
+        """
+        target: test search with expressions with double quotes
+        method: test search with expressions with double quotes
+        expected: searched successfully with correct limit(topK)
+        """
+        # 1. initialize with data
+        collection_w = self.init_collection_general(prefix)[0]
+        string_value = [f"{cf.gen_str_by_length()}\"{cf.gen_str_by_length()}\"" for _ in range(default_nb)]
+        data = cf.gen_default_dataframe_data()
+        data[default_string_field_name] = string_value
+        insert_ids = data[default_int64_field_name]
+        collection_w.insert(data)
+
+        # 2. create index
+        index_param = {"index_type": "FLAT", "metric_type": "COSINE", "params": {}}
+        collection_w.create_index("float_vector", index_param)
+        collection_w.load()
+
+        # 3. search with expression
+        _id = random.randint(0, default_nb)
+        string_value[_id] = string_value[_id].replace("\"", "\\\"")
+        expression = f"{default_string_field_name} == \"{string_value[_id]}\""
```

Comment:
shall we also add more scenarios for multiple single/double quotes in the inserted string

Reply:
added.